### ISSUE TITLE
Android 5.0.0-dev testing needed, .apk linked

### ISSUE THREAD

**Issue by paramat** [minetest/minetest#7965]
There has been a lot of work on the Android app recently and it is very important that the app is the best MT app available when 5.0.0 is released (partly because MultiCraft development will likely end so there may be a very large number of players looking to move to another app).
stujones11 is providing regular updated .apk's in the forum here https://forum.minetest.net/viewtopic.php?f=42&t=21526
Please test.

**Comment by nerzhul** [comment]
I have the formal official APK on my machine i can provide it in a github prerelease this evening if needed.
I tested it and it works like a charm on my phone and it's better than previous version.

**Comment by nerzhul** [comment]
i don't think it's a blocker issue, because there is no issue, and i already tested it as i said :)

**Comment by nerzhul** [comment]
My build is based on 56f22bfa5c22a89db968cf2dfc4d070b5f0e6520

i should update to latest because we have two minor changes

**Comment by stujones11** [comment]
> i can provide it in a github prerelease

This should hopefully get more attention since you can actually sign it. However, I will continue to provide my unofficial builds, maybe extend them to PR testing, etc if there will be a regular official one.

**Comment by Fixer-007** [comment]
Does it support 4.4.2?

**Comment by Fixer-007** [comment]
@nerzhul can you post a link to your builds? Also do both support KitKat?

**Comment by paramat** [comment]
Yes this isn't really a blocker as testing is already known to be necessary before a release.

**Comment by niansa** [comment]
My friend just gets: "App not installed."

**Comment by paramat** [comment]
nerzhul's latest build still has bugs.

**Comment by niansa** [comment]
rubenwardy: He told me that he did that.

**Comment by Extex101** [comment]
Tested:
1. Crashes with no reason while in content screen
2. Server joining is unexpectedly slow
3. Content-store is also very limited

**Comment by niansa** [comment]
I tested on my own device (Samsung Galaxy Fame):

It just crashes my entire device with a blue screen (not a bluescreen, but a blue screen). That happens before any gui things do appear.

**Comment by stujones11** [comment]
@niansa The blue screen would suggest that the app did actually try to start but failed to launch the client. This can be due to conflicts with earlier versions of the game files that to not automatically get removed when you uninstall from the app manager.

@Extex101 

1. By content screen, I assume that you mean content store. If so, does this occur frequently?
2. This is to be expected, it should be much quicker the next time you join.
3. Please elaborate, it _should_ have exactly the same features as the pc version.

**Comment by Extex101** [comment]
Yes the whole app crashes
It makes it extremely hard to use

K so I launched the app, saw tapped the "Content" button, then pressed "Browse online content", it didn't work when I pressed it so I pressed again, then tried to go back to the main page, then it flicked to the online content page (duration about 15 seconds) then i went mod shopping for a minute or two then it crashed

**Comment by stujones11** [comment]
@Extex101 My guess is that you will need to be a little more patient with the content store, although I admit that flicking between menus should not cause a crash.

**Comment by Extex101** [comment]
"More patient" meaning
Deal with the constant crashes while trying to look at the mods? 
Anywho I uninstalled it because of constant crashes

**Comment by niansa** [comment]
But can you make a fix that does not crash my entire device?

**Comment by rubenwardy** [comment]
@Extex101 the content crashes have been fixed now. It was nothing to do with that page, but a bug that's existed since 2014 but the crash is made faster by there being so many images

**Comment by stujones11** [comment]
My dev build has now been updated with the latest fixes: https://forum.minetest.net/viewtopic.php?f=42&t=21526

**Comment by PorygonZRocks** [comment]
Using stu's latest build (on commit badecfa), the app freezes while downloading content from the store and gives an "app not responding" message. If you hit "wait" on the message it downloads everything fine eventually, (sometimes after hitting wait on several more messages) but users might end up closing the app because of the message appearing so much.

**Comment by jastevenson303** [comment]
Though vague, here is a "report" from a user on Reddit:
https://www.reddit.com/r/Minetest/comments/ah3cl1/anyone_else_having_trouble_playing_this_in/

**Comment by paramat** [comment]
^ But that's probably 0.4.17.1-3 which still has the GUI scaling issues.

**Comment by jastevenson303** [comment]
Okay good point, thank you.  I referred him here to possibly test new version.

**Comment by Innomen** [comment]
> Though vague, here is a "report" from a user on Reddit:
> ^ But that's probably 0.4.17.1-3 which still has the GUI scaling issues.

That is my report, sorry I didn't include the version, just slipped my mind.

Indeed it's 0.4.17.21 from fdroid.

Thank you both :)

Unfortunately I have installed the new one and still have the same issue.

![screenshot_20190120-011741](https://user-images.githubusercontent.com/134324/51435957-5970bc80-1c51-11e9-81c0-2df420e493a8.png)

**Comment by stujones11** [comment]
Maybe this can be closed now?

I will continue to release my unofficial dev builds, I just didn't want to cause any confusion with an official RC build that never even happened, anyway =)

**Comment by Innomen** [comment]
Dude... It's still broken. I said a month ago I tested the new one and this issue persists. It persists in the releases apk from here also. I'll post a new report I guess.

**Comment by paramat** [comment]
Thanks, a new issue is best yes.

**Comment by stujones11** [comment]
> Dude... It's still broken.

Try deleting `minetest.conf` if that doesn't fix it then file a new issue and I will look into it.

**Comment by Innomen** [comment]
> > Dude... It's still broken.
> 
> Try deleting `minetest.conf` if that doesn't fix it then file a new issue and I will look into it.

I forgot to update this, my bad. It was indeed something in a folder that needed to be deleted, also I had another app that was allowing me to force rotation into portrait which isn't actually supported by minetest.

I just didn't even think of treating android like windows and look for leftovers after an uninstall XD

Thanks for your help all :)

**Comment by niansa** [comment]
@stujones11
> **Update:** This issue seems to be with android 4.x which the galaxy fame is using. I checked the specs on it and would be mighty impressed if MT were to be at all playable on that :)

The 0.1x.x ran fine
